Equilobe.DailyReport.Models/Interfaces/IDataService.cs

```diff
@@ -19,5 +19,6 @@ public interface IDataService : IService
         System.Collections.Generic.List<string> GetUniqueProjectsKey(string baseUrl);
         void SetReportFromDb(Equilobe.DailyReport.Models.ReportFrame.JiraReport report);
         JiraPolicy GetPolicy(string uniqueProjectKey);
+        byte[] GetImageFromDb(string key);
```

Comment:
Toate chestiile vind dintr-un storage (fisiere, db, etc.). Nu e cazul ca aceste metode sa aiba asta in nume.

Deci metoda ar trebui sa fie GetImage sau GetImageByKey.